snow/engine/common/bootstrapper.go

```diff
@@ -307,7 +325,7 @@ func (b *bootstrapper) Startup(ctx context.Context) error {
 	b.acceptedFrontierSet.Clear()
 
 	b.pendingSendAccepted.Clear()
-	for nodeID := range b.Beacons.Map() {
+	for nodeID := range b.Beacons.GetMap(b.Ctx.SubnetID) {
```

Comment:
nit: replace `GetMap` with `GetValidatorIDs(subnetID ids.ID)`. It should be a bit more efficient, avoding all copies

Reply:
good idea